os/include/tinyara/fs/ioctl.h

```diff
@@ -161,6 +161,11 @@
 										 * OUT: Bytes writable to this fd
 										 */
 
+#define FIONBIO         _FIOC(0x000b)     /* IN:  Boolean option takes an
+                                           *      int value.
+                                           * OUT: Origin option.
```

Comment:
`OUT: Origin option.`
What exactly is meant by "Origin option"?
Do you mean _Original option_?

Reply:
yes you can say it, means base.